doc/syslog/syslog-design.md

```diff
@@ -503,12 +753,14 @@ module sonic-syslog {
 
                 leaf server_address {
                     description "Syslog server IP address";
-                    type inet:ip-address;
+                    type inet:host;
```

Comment:
By this change it also makes the ip address that can be configured using domain name. Is that correct? Please update other locations in this file too